x/ibc-hooks/testutils/contracts/echo/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "echo"
+description = "Cosmwasm contract that always returns the same response"
```

Comment:
Where is this contract used? I think I only saw counter in tests?

Reply:
The old tests use that contract: https://github.com/osmosis-labs/osmosis/blob/dee352ec296f673ffc63feafad7f883022460435/x/ibc-hooks/ibc_middleware_test.go#L191-L204 

I had just added the bytecode before but not the source. Added the source now since we're adding the counter anyway